.github/workflows/ci.yml

```
@@ -10,8 +10,8 @@ jobs:
       max-parallel: 10
       fail-fast: false
       matrix:
-        php: ['7.3', '7.4', '8.0']
-        sf_version: ['4.4.*', '5.0.*', '5.2.*']
+        php: ['7.3', '7.4', '8.0', '8.1']
+        sf_version: ['4.4.*', '5.0.*', '5.2.*', '5.4.*', '6.0.*']
```

Symfony 6 is compatible with php >= 8.0 so running tests on 7.3 and 7.4 will always fail

```suggestion
        sf_version: ['4.4.*', '5.0.*', '5.2.*', '5.4.*', '6.0.*']
        exclude:
          - php: 7.3
            sf_version: 6.0.*
            
          - php: 7.4
            sf_version: 6.0.*
```